sympy/utilities/tests/test_wester.py

```diff
@@ -987,7 +987,7 @@ def test_M23():
 def test_M24():
     # TODO: Replace solve with solveset, as of now test fails for solveset
     solution = solve(1 - binomial(m, 2)*2**k, k)
```

Comment:
Can we get the original solution with an assumption on m?

Reply:
Yupp!